src/constants.js

```diff
@@ -16,6 +16,25 @@ exports.DSTU2 = {
 	}
 };
 
+exports.STU3 = {
+	RESOURCE_TYPES: {
+		OPERATIONOUTCOME: 'OperationOutcome',
+		CONFORMANCE: 'CapabilityStatement',
+		OBSERVATION: 'Observation',
+		PATIENT: 'Patient'
```

Comment:
I don't like having to define these here.  I don't want to have to auto-generate the elements in this file to match the auto-generated classes.